tests/cli_tests.py

```diff
@@ -57,3 +60,41 @@ def test_import_controlled_list(self):
         self.assertTrue(List.objects.filter(pk=list_pk).exists())
 
     ### TODO Add test for creating new language if language code not in db but found in import file
+
```

Comment:
I missed this before, but in input_file above, avoid providing a slashed input as one of the args to os.path.join, which could fail on windows. Provide each path component as separate args to join().